App instantly crash using 9.1.0

I resolved by including 
`implementation("com.google.guava:guava:33.3.1-jre")`

in the build.gradle, if needed should be added on the release upgrade changes?

Okay, reproduced on the crash - sorry again for that:

```
10-07 08:33:35.570  6839  6839 E AndroidRuntime: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/common/util/concurrent/MoreExecutors;
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at app.notifee.core.Notifee.<clinit>(SourceFile:3)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at io.invertase.notifee.NotifeeInitProvider.onCreate(NotifeeInitProvider.java:20)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2619)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2589)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at app.notifee.core.InitProvider.attachInfo(SourceFile:7)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.app.ActivityThread.installProvider(ActivityThread.java:7765)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:7276)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6983)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2236)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:205)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:294)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8177)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.common.util.concurrent.MoreExecutors" on path: DexPathList[[zip file "/data/app/~~4rC8IOrbR_iIvM82Fq311g==/com.notifeedemo-2DRFDfVKDL9n5A73DQ7j8g==/base.apk"],nativeLibraryDirectories=[/data/app/~~4rC8IOrbR_iIvM82Fq311g==/com.notifeedemo-2DRFDfVKDL9n5A73DQ7j8g==/lib/arm64, /data/app/~~4rC8IOrbR_iIvM82Fq311g==/com.notifeedemo-2DRFDfVKDL9n5A73DQ7j8g==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
10-07 08:33:35.570  6839  6839 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:637)

```

This points to a couple issues, one is this crash itself, which I confirm is fixed, merely by adding this to the module's build.gradle (directly, and/or via patch-package, or to your app's build.gradle but that shouldn't be necessary):

```groovy
  implementation("com.google.guava:guava:33.3.1-android") // https://github.com/google/guava
```

second issue is that nothing like a startup crash should ever slip through CI and as a secondary thing I'll make sure this can't happen again.

Release with a fix coming shortly and thanks again for reporting

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -108,7 +108,7 @@ dependencies {
   api 'androidx.work:work-runtime:2.8.0' // https://developer.android.com/jetpack/androidx/releases/work
   api 'com.facebook.fresco:fresco:2.6.0' // https://github.com/facebook/fresco/releases
 
-  implementation("com.google.guava:guava:31.1-android") // https://github.com/google/guava
+  implementation("com.google.guava:guava:33.3.1-android") // https://github.com/google/guava
   implementation 'androidx.core:core:1.6.0'
 
   def room_version = '2.5.0' // https://developer.android.com/jetpack/androidx/releases/room

--- a/packages/react-native/android/build.gradle
+++ b/packages/react-native/android/build.gradle
@@ -98,6 +98,8 @@ dependencies {
   } else {
     implementation(group: 'app.notifee', name:'core', version: '+')
   }
+
+  implementation("com.google.guava:guava:33.3.1-android") // https://github.com/google/guava
   implementation 'androidx.concurrent:concurrent-futures:1.1.0' // https://developer.android.com/jetpack/androidx/releases/concurrent
   implementation 'androidx.work:work-runtime:2.8.0' // https://developer.android.com/jetpack/androidx/releases/work
   implementation 'org.greenrobot:eventbus:3.3.1' // https://github.com/greenrobot/EventBus/releases

